app/components/section-work.hbs

```diff
@@ -16,7 +16,8 @@
             <div class="form-group">
               <fieldset>
                 <legend class="input-label">
-                  Do you currently use Ember at your place of work?
+                  Does your workplace currently use Ember for any of its operations?
+                  Note: you do not have to personally use Ember at work to answer 'Yes' to this question.
```

Comment:
Do you think you should convert the note to an aria-describedby paragraph for consistency with the rest of the elements? like this: 

```
<legend class="input-label" aria-describedby="work-use-desc">Does your workplace currently use Ember for any of its applications?</legend>
<p id="work-use-desc" class="w-full mt-0 mb-2 text-sm text-slate-700">Note: you do not have to personally work on the Ember app to answer 'Yes' to this question.</p>
```

WDYT?

Reply:
Also, if they don't have to use Ember to answer yes, should the question be changed to 

"Is Ember used in any applications at the place where you work?"

Reply:
> Do you think you should convert the note to an aria-describedby paragraph for consistency with the rest of the elements? like this:
> 
> ```
> <legend class="input-label" aria-describedby="work-use-desc">Does your workplace currently use Ember for any of its applications?</legend>
> <p id="work-use-desc" class="w-full mt-0 mb-2 text-sm text-slate-700">Note: you do not have to personally work on the Ember app to answer 'Yes' to this question.</p>
> ```
> 
> WDYT?

- Agree with the `<p>` structure
- Agree with'operations' -> 'applications' in question
- Not sure 'the Ember app' is the right phrasing -- 'personally work on an Ember app for your job / at your place of work'